configuration_files/hsr.lua

```diff
@@ -21,7 +21,7 @@ options = {
   published_frame = "odom",
   odom_frame = "odom",
   provide_odom_frame = false,
-  use_odometry_data = true,
+  use_odometry_data = false,
```

Comment:
Because you get a better map of your building this way.

Longer version: the current configuration isn't using the IMU, so it's relying on the odometer. There's a spot where the robot hits a curb, which bungles the odometry, which leads to a bad submap, which leads to some doubling in the result. An IMU would likely help, but in the meantime, disabling the odometry lets it rely only on the laser, and that works quite well.